README.md

```diff
@@ -81,6 +81,14 @@ yolox "extract audio from maths.mp4 and save it as maths.m4a" --model=llama3
 # ffmpeg -i maths.mp4 -vn -acodec copy maths.m4a
 ```
 
+### Print mode
+
+Print the command but don't execute it:
+
+```console
+yolox --print "extract audio from maths.mp4 and save it as maths.m4a"
+# ffmpeg -i maths.mp4 -vn -acodec copy maths.m4a
```

Comment:
I'm into this idea!

But lets change the output to not have a `#` comment at the start. That way you can copy the full output, or pipe it into a file, or pipe it into bash, without having the tweak it.

Reply:
It already doesn't include `#`, I just thought to make the documentation consistent with other examples. But you're right that here it's misleading 👍